source/geometry_model/initial_topography_model/prm_polygon.cc

```diff
@@ -100,6 +110,7 @@ namespace aspect
              * we need to fill the point lists and topography values. They
              * are stored in the Topography subsection in the Topography parameter.
              */
+            maximum_topography = 0;
```

Comment:
In the hypothetical case that all prescribed topography is negative, taking the maximum with zero won't see the topography. Not sure how that affects anything down the line, but I guess it would be cleanest to use -std::numeric_limits<double>::max() for example.